PhysicsTools/PatAlgos/python/producersHeavyIons/heavyIonJets_cff.py

```diff
@@ -15,13 +15,13 @@
     src = 'allPartons',
     )
 
-ak4HiCleanedGenJets = heavyIonCleanedGenJets.clone(src = "ak4HiGenJets")
+ak4HiGenJetsCleaned = heavyIonCleanedGenJets.clone(src = "ak4HiGenJets")
```

Comment:
I thought that this was already merged in the master

Reply:
Good eye.  Should I revert the file to the previous commit?

Reply:
If you rebase these updates in this file will probably disappear from the difference.
On the other hand, it is just a couple of lines in one single file, github does not complain about possible conflicts: we can even let it as such, here